### DISCUSSION TITLE
How many workers can a tasklist associated?

### DISCUSSION THREAD

**Discussion by zhangkyou** [uber/cadence#4401] [Q&A] [answered]
In one worker, the worker only associate to the tasklist of groupName.
`worker := worker.New(h.Service, domainName, groupName, options)`

But if I had a cluster to deploy the cadence worker, every machine in the cluster run the same worker code, so there had  many workers associate to the same tasklist, does this deployment matter to the workflow schedule?

In other words,can several workers polling task from one tasklist parallel?

**Comment by longquanzheng** [comment, marked as answer]
There is no limit on that.

**Comment by meiliang86** [comment]
Hundreds of workers should be fine. If you want more than a few thousands of workers, then you might want to enable scalable task list.
